### PR TITLE
[FIX] maxFee: save settings to userBlob (RT-3289)

### PR DESCRIPTION
**Pull request by rht**

### CODE REVIEW

File: src/js/tabs/advanced.controller.js [hunk 1]
```diff
@@ -67,6 +67,7 @@ AdvancedTab.prototype.angular = function(module)
           // This has to be updated manually because the network object is not
           // recreated unless we do location.reload()
           network.remote.max_fee = $scope.options.max_tx_network_fee;
+          $scope.userBlob.set('/clients/rippletradecom/maxNetworkFee', $scope.options.max_tx_network_fee)
```
Comment (yongsoo) [inline src/js/tabs/advanced.controller.js]: @vhpoet Are we maintaining these in blobvault anymore?

Comment (vhpoet) [inline src/js/tabs/advanced.controller.js]: @yongsoo why not?

**Comment by mrajvanshy** [comment]
Should we make this PR to release?

File: src/js/tabs/advanced.controller.js (hunk 1 above)
Comment (yongsoo) [inline src/js/tabs/advanced.controller.js]: Ok just checking. Wasn't sure if we were changing blobvault with our identity program.

**Comment by annatonger** [comment]
yes, @rht  please close and make this to Release instead. thanks!

**Pull request closed without merging**